concent_api/core/payments/mock.py

```diff
@@ -33,43 +18,20 @@ def get_list_of_transactions(_oldest_payments_ts = None, current_time = None, _t
             payment_ts = current_time - settings.PAYMENT_DUE_TIME
             return [{'timestamp': payment_ts - 3700}, {'timestamp': payment_ts - 3800}, {'timestamp': payment_ts - 3900}]
         else:
-            return [{'timestamp': current_time - 22}, {'timestamp': current_time - 23}, {'timestamp': current_time - 24}]
-    return True
-
-
-def get_forced_payments(_oldest_payments_ts = None, _requestor_address = None, _provider_address = None, _to_block = None, request = None, current_time = None):  # pylint: disable=inconsistent-return-statements
-    """
-    Function which return list of forced paysments from payment API
-    where t0 <= payment_ts + PAYMENT_DUE_TIME
-    """
-
-    if 'HTTP_TEMPORARY_LIST_OF_FORCED_TRANSACTIONS' in request.META:
-        if bool(request.META['HTTP_TEMPORARY_LIST_OF_FORCED_TRANSACTIONS']):
-            return [{'timestamp': current_time - 3700}, {'timestamp': current_time - 3800}, {'timestamp': current_time - 3900}]
-        else:
-            return [{'timestamp': current_time - 22}, {'timestamp': current_time - 23}, {'timestamp': current_time - 24}]
+            return [{'closure_time': current_time - 22}, {'closure_time': current_time - 23}, {'closure_time': current_time - 24}]
     return True
 
 
-def make_payment_to_provider(_sum_of_payments = None, _payment_ts = None, _requestor_ethereum_public_key = None, _provider_ethereum_public_key = None):
+def make_force_payment_to_provider(_concent_rpc = None, _requestor_eth_address = None, _provider_eth_address = None, _value = None, _payment_ts = None):
     """
     Concent makes transaction from requestor's deposit to provider's account on amount V.
     If there is less then V on requestor's deposit, Concent transfers as much as possible.
     """
-    return Decimal('9.99')
-
+    return 100
 
-def make_forced_payment(_provider = None, _requestor = None):
 
-    return True
-
-
-def is_provider_account_status_positive(request = None):
+def is_provider_account_status_positive(request = None, _concent_rpc = None, _requestor_eth_address = None, _pending_value = None):
     if 'HTTP_TEMPORARY_ACCOUNT_FUNDS' in request.META:
```

Comment:
Same here. @dybi

Reply:
this one is being used in some of tests for `SubtaskResultsVerify` cases

Reply:
As above I change mock backend to return null values